databases/week3/hyf_lesson3.sql

```diff
@@ -0,0 +1,196 @@
+CREATE DATABASE hyf_lesson3;
```

Comment:
When i try to run the code here i get a bunch of errors, please fix it so i can run it without problems :)